tests/tracking/test_client.py

```diff
@@ -252,3 +253,63 @@ def test_registry_uri_from_implicit_tracking_uri():
         get_tracking_uri_mock.return_value = tracking_uri
         client = MlflowClient()
         assert client._registry_uri == tracking_uri
+
+
+def test_create_model_version_normal(mock_registry_store):
```

Comment:
`test_create_model_version_nondatabricks_source_no_runlink` maybe? should also add a test with `run_link` explicitly provided to `MlflowClient.create_model_version`.